src/Chainweb/Chainweb.hs

```diff
@@ -616,9 +617,10 @@ runChainweb
     -> IO ()
 runChainweb cw = do
     logg Info "start chainweb node"
+    now <- getCurrentTimeIntegral
```

Comment:
What about moving this right before `mempoolsToServe` in line 652? That would avoid having to pass the parameter around (something that we'll have to undo later).

Reply:
To avoid forcing `mempoolsToServe` into `IO`, which would have to be undone in future anyway.